frontend/shared/browser-tests/reporter/utils/emojis.ts

```diff
@@ -0,0 +1,12 @@
+export const emojis = {
+  rocket: ':rocket:',
```

Comment:
should we use UPPER_CASE for enum consts?

Reply:
fixed

Reply:
thx